docs/docs/09-faqs.md

```diff
@@ -61,3 +61,16 @@ If you're already authenticated with the `gh` CLI, you can use its token by runn
 ```bash
 export GITHUB_AUTH_TOKEN="$(gh auth token)"
 ```
+
+### I'm not able to get GPTScript to work with my OpenAI account which is in "Free Trial" 
```

Comment:
Ive added some suggestions to the text targeted at answering this exact question, but really there are sort of 2 scenarios that could lead to the '404 model not found' error. It might be worth splitting this into two sections, one that directly addresses the question of what to do if you get that error, and the other that addresses the rate limit error.

I think people will more frequently be asking questions about those errors they see directly from gptscript, where for now this is question is working backwards from an exact answer.